cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,58 @@
 /// <reference types='cypress' />
-
 describe('Web Tables page', () => {
-  it('', () => {
+  
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+   
+
+  it('should check pagination', () => {
+    cy.generate_N_Users(10);
+    cy.contains('.-btn', 'Next').click();
+    cy.get('[aria-label="jump to page"]').should('have.value',
+     '2', {timeout: 10000});
+  });
+
+  it('should check rows count selection', () => {
```

Comment:
add an assertion for table rows count, because now you just check the number in dropdown

Reply:
still not fixed. You check the empty space 
![image](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/e5c3f26c-5455-41ef-9a1c-83dfa794ffb8)